testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/deployment/moduleslisting/DeploymentModulesListTestCase.java

```diff
@@ -25,10 +25,12 @@
 import java.util.ArrayList;
 import java.util.List;
 import java.util.stream.Collectors;
+import integration.deployment.moduleslisting.UserModuleSetupTask;
```

Comment:
@khermano Please use the fully qualified class name, not just the relative part.

After ~ 25 years working with Java, I had no idea you could do this!

Reply:
Ah, I suspect you can't do that. This doesn't fail because the UserModuleSetupTask that gets used is in the same package and doesn't require an import. So the incorrect import statement is never used. Please remove it.

Reply:
I am sorry. I accidentally removed the beginning of the package path in the UserModuleSetupTask, which is why such an import exists.
I fixed it. Thank you for the review.